.github/workflows/test-signer-darwin.yml

```diff
@@ -45,3 +45,11 @@ jobs:
       with:
         name: darwin_arm64
         path: ./build/bin/darwin_arm64/*
+
+  cred:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Get test credential
+      run: scripts/keychain_cred.sh
```

Comment:
This should be the first step of the existing job

Reply:
Since I merged my cred job into the existing job (build), should I move the script (keychain_cred.sh) to be in the build folder?

Reply:
No let's leave it in the scripts folder